async/async-and-await/cs/ParallelAsyncExample/MainWindow.xaml.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Threading;
+
+namespace ParallelAsyncExample
+{
+    public partial class MainWindow : Window
+    {
+        readonly HttpClient _client = new HttpClient { MaxResponseContentBufferSize = 1_000_000 };
+
+        readonly IEnumerable<string> _urlList = new string[]
+        {
+            "https://docs.microsoft.com",
+            "https://docs.microsoft.com/azure",
+            "https://docs.microsoft.com/powershell",
+            "https://docs.microsoft.com/dotnet",
+            "https://docs.microsoft.com/aspnet/core",
+            "https://docs.microsoft.com/windows",
+            "https://docs.microsoft.com/office",
+            "https://docs.microsoft.com/enterprise-mobility-security",
+            "https://docs.microsoft.com/visualstudio",
+            "https://docs.microsoft.com/microsoft-365",
+            "https://docs.microsoft.com/sql",
+            "https://docs.microsoft.com/dynamics365",
+            "https://docs.microsoft.com/surface",
+            "https://docs.microsoft.com/xamarin",
+            "https://docs.microsoft.com/azure/devops",
+            "https://docs.microsoft.com/system-center",
+            "https://docs.microsoft.com/graph",
+            "https://docs.microsoft.com/education",
+            "https://docs.microsoft.com/gaming",
+        };
+
+        void OnStartButtonClick(object sender, RoutedEventArgs e)
```

Comment:
Access modifiers are missing to all methods

Reply:
They're not missing, they were intentionally omitted. I prefer to omit default access modifiers as I believe it makes it easier to read, and they're redundant. But, I added them.